pkg/api/install/install.go

```diff
@@ -89,7 +89,8 @@ func enableVersions(externalVersions []unversioned.GroupVersion) error {
 }
 
 // userResources is a group of resources mostly used by a kubectl user
-var userResources = []string{"rc", "svc", "pods", "pvc"}
+var userResources = []string{"cs", "configmaps", "ds", "deploy", "ep", "hpa", "ing", "jobs",
```

Comment:
Secrets are potentially unrecoverable, so like PVC they are dangerous to
make automatic.  If we do that for secrets config maps should follow the
same rules.

On Thu, Jul 14, 2016 at 12:28 PM, David Eads <notifications@github.com>
wrote:

> In pkg/api/install/install.go
> <https://github.com/kubernetes/kubernetes/pull/28955#discussion_r70837452>
> :
>
> > @@ -89,7 +89,8 @@ func enableVersions(externalVersions []unversioned.GroupVersion) error {
> >  }
> >
> >  // userResources is a group of resources mostly used by a kubectl user
> > -var userResources = []string{"rc", "svc", "pods", "pvc"}
> > +var userResources = []string{"cs", "configmaps", "ds", "deploy", "ep", "hpa", "ing", "jobs",
>
> no component status, no endpoints (not an 80% case), no limits, no nodes,
> keep pods spelled out for people.
>
> @smarterclayton <https://github.com/smarterclayton> I'm on the fence for
> secrets and configmaps, you?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/28955/files/982c15f0c8e4b30fc31430df19d5c6dd6723b8b9#r70837452>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABG_p7hodLOCqfDJlgWI1yeBCeY_8vExks5qVmO9gaJpZM4JMW31>
> .
>